tests/functional/b/bad_typevar_name.py

```diff
@@ -0,0 +1,23 @@
+"""Test case for bad-typevar-name."""
+
+from typing import TypeVar
+
+T = TypeVar("T")
+T_co = TypeVar("T_co")  # [bad-typevar-name]
+T_contra = TypeVar("T_contra")  # [bad-typevar-name]
+
+CT_co = TypeVar("CT_co", contravariant=True)  # [bad-typevar-name]
+RT_contra = TypeVar("RT_contra")  # [bad-typevar-name]
```

Comment:
Because that variable is _invariant_ but the name is using suffix contra. _invariant_ variable should not uses suffixes contra and co as it will cause confusing.

Reply:
Ok. Thanks.